src/content/docs/data-apis/manage-data/manage-data-retention.mdx

```diff
@@ -688,7 +688,7 @@ To view and edit your retention, go to [the data management hub](https://one.new
 Options for increasing retention: 
 
 * If you haven't already, switch to [Data Plus](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/data-ingest-billing/#data-plus), which offers significantly longer data retention and other benefits.
-* For Data Plus or original data option: add data retention. 30 days of additional data retention for your organization costs $0.05 per GB ingested per 30 days.
+* For Data Plus or original data option: add data retention. 30 days of additional data retention for your organization costs $0.05 per GB ingested per month.
```

Comment:
@zuluecho9 the phrase "add data retention" following the colon wasn't quite clear to me. I know you didn't change this part of the sentence now, but you might consider something like this:

For Data Plus or original data option, you can add 30 days of additional data retention in your organization for $0.05 per GB ingested per month.